Automattic/newspack-plugin
# SEO Wizard/Yoast Configuration Manager

### All Submissions:






### Changes proposed in this Pull Request:

Replaces the SEO Wizard with four screens that set and configure a few key Yoast SEO settings.

Closes https://github.com/Automattic/newspack-plugin/issues/425

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Navigation to Newspack > SEO
3. Change the settings
4. Check directly in Yoast if the settings have been changed properly.

### Other information:
